opennms-webapp/src/main/java/org/opennms/web/event/EventQueryServlet.java

```diff
@@ -133,6 +135,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
             filterArray.add(new ExactUEIFilter(exactUEI));
         }
 
+        // convenient syntax for LocationFilter
+        String location = WebSecurityUtils.sanitizeString(request.getParameter("location"));
+        if (location != null && !location.equalsIgnoreCase("any")) {
```

Comment:
This could be simplified by reverting the equals: `"any".equalsIgnoreCase(location)`.